site/src/pages/WorkspacePage/WorkspaceReadyPage.tsx

```diff
@@ -100,13 +100,11 @@ export const WorkspaceReadyPage = ({
       ["canceling", "deleting", "pending", "starting", "stopping"].includes(
```

Comment:
1. User does not know which parameters are ephemeral - I mentioned this in the definition of done.
2. Now, "Parameters" form includes ephemeral/build options in every request. We don't want this behavior to happen, and we don't want to propagate options between consecutive builds.

<img width="1385" alt="Screenshot 2023-07-17 at 10 44 47" src="https://github.com/coder/coder/assets/14044910/6521c1a5-b00a-433a-957a-e95d201a20e0">

Reply:
Currently, all parameters including ephemeral ones, are in the same table, which means that they will be included in the next workspace build request.

How user can prevent it from happening?